examples/many_views.tf

```diff
@@ -2,103 +2,22 @@ provider "logdna" {
   servicekey = "Your service key goes here"
 }
 
-resource "logdna_view" "my_view" {
-  name  = "test1"
-  query = "test"
+resource "logdna_view" "kube_probes" {
+  name  = "kube_probes"
+  query = "agent:kube-probe"
 }
 
-resource "logdna_view" "my_view2" {
-  name  = "test2"
-  query = "test"
+resource "logdna_view" "e2e_metrics_reporter" {
+  name  = "e2e_metrics_reporter"
+  query = "app:e2e-latency-metrics-reporter"
 }
 
-resource "logdna_view" "my_view3" {
-  name  = "test3"
-  query = "test"
+resource "logdna_view" "not_info_logs" {
+  name  = "not_info_logs"
+  query = "-level:info"
 }
 
-resource "logdna_view" "my_view4" {
-  name  = "test4"
-  query = "test"
+resource "logdna_view" "exception_errors" {
+  name  = "exception_errors"
+  query = "ClassCastException OR NullPointerException"
```

Comment:
Some of these examples should use more properties than just name/query.  It's a little redundant otherwise.

Reply:
fixed